react/index.d.ts

```diff
@@ -606,165 +606,173 @@ declare namespace React {
         onTransitionEndCapture?: TransitionEventHandler;
     }
 
+    // See CSS 3 Explicit Defaulting https://www.w3.org/TR/css-cascade-3/#defaulting-keywords
+    // "all CSS properties can accept these values"
+    type ExplicitDefaultingCSSKeywords = "initial" | "inherit" | "unset";
```

Comment:
can we use the name `CSSWideKeyword` instead? based on the spec at https://www.w3.org/TR/css3-values/#common-keywords